Continuation of PR#2

I2pRandom opened the pull request:
Added 3 items, originally the first as a joke. but continued to clean it up and eventually we got this now..
Added a Buttplug, a Large Buttplug for those with just a "little more room", and finally a Fox Tail Buttplug. 
I'm also tempted to add the Fancy tag to the Fox tail, and give the Fox tail able to use reach attacks (imagine holding it by the tail end and swinging it around wildly as a weapon). 

Note idk how to tag a previous PR, so lets hope first time is the charm.

I2pRandom committed:
Adding joke item

Added Joke item Buttplug

I2pRandom committed:
updating Buttplug

fixing slight discrepancy in description.

I2pRandom committed:
another update to the buttplug item.

this joke has gone beyond a joke to an actual item addition, cus fuck it.
Added the "ONLY_ONE" flag, Removed the cover tag so that it's 'socketless' with (ty Kheir), and finally redefined the description for one that feels better.
still no idea how to apply the 'fun' artifact effect to the item as a stand-in for a custom morale bonus.

I2pRandom committed:
correct dead encumberance stat.

I2pRandom committed:
update to misc.json

Tweaks to the buttplug item, and additions of a Fox tail plug, and a large plug.

I2pRandom committed:
Merge branch 'master' of https://github.com/I2pRandom/Cataclysm-DF

I2pRandom commented:
ok i need to learn github better, it seems my previous PR edits are included in this PR. so it's best to actually keep 1 single PR and just keep doing edits to it, and mainly edit on my end then submit them to keep the commits reasonable.

ChainsawGarden reviewed (changes requested):
Looks good! Could you [lint](http://dev.narc.ro/cataclysm/format.html) those three JSON entries and add periods to the three items descriptions?

You might also want to add the appropriate spawns to homes too.

Thanks for being the first contributor to Delta Fork's JSON-sphere!

I2pRandom committed:
Cleaning up Json 

Found some entry duplicates, seems i hit Ctrl+V at some point as i was going through it. I do not know how to add house spawns, and i don't understand this lint tool. it just gives me syntax errors, even those these items are pretty much a straight copy of bead item in the jewelry.json

ChainsawGarden reviewed (approved):
Everything looks great, I'll handle it from here. I appreciate your contribution!
